src/utils/keyboard-interceptor.ts

```diff
@@ -10,43 +12,66 @@ export const KEYCODES = {
 };
 
 export default class KeyboardInterceptorCore {
-  private config;
+  private _eventEmitter;
+  private _node;
+
+  constructor(node, callbacks?) {
+    this._eventEmitter = new EventEmitter();
+    this._node = node;
 
-  constructor(config) {
-    this.config = config;
+    callbacks && this._attachCallbacks(callbacks);
     this._bindCallbacks();
     this._bindEvents();
   }
 
+  _attachCallbacks(callbacks) {
+    Object.keys(callbacks).forEach(keyCode => {
+      const keyCodeCallbacks = callbacks[keyCode];
+      if (Array.isArray(keyCodeCallbacks)) {
+        keyCodeCallbacks.forEach(callback =>
+          this._eventEmitter.on(keyCode, callback),
+        );
+      } else {
+        this._eventEmitter.on(keyCode, keyCodeCallbacks);
+      }
+    });
+  }
+
+  _unattachCallbacks() {
+    this._eventEmitter.eventNames().forEach(eventName => {
```

Comment:
mb `this._eventEmitter.removeAllListeners();` will be enougth

https://nodejs.org/api/events.html#events_emitter_removealllisteners_eventname\